### ISSUE TITLE
"Edit in Wiki" link/button not doing anything

### ISSUE THREAD

**Issue by peterbe** [mdn/kuma#6492]
I've only tested this in FirefoxNightly on macOS. But happens even if I refresh the page. 

Sample URL: https://developer.mozilla.org/en-US/docs/MDN/Jobs
(you have to be logged in for the "Edit in wiki" button to appear)

The presence of the button only appears when you're signed in. 
Clicking it does nothing. I can't even tell if there's an onclick event handler attached to it because perhaps there's a onclick even handler to its parent (or greatgreat grand parent) that filters based on target. 

![Screen-Recording-2020-02-13-at-11 53 39-AM](https://user-images.githubusercontent.com/26739/74459569-8776ff00-4e59-11ea-9746-3c062b606714.gif)

<img width="1504" alt="Screen Shot 2020-02-13 at 12 06 06 PM" src="https://user-images.githubusercontent.com/26739/74459587-8fcf3a00-4e59-11ea-88e1-56593f745814.png">

Basically, the only way for me to get to the `wiki.` equivalent of the URL is if I right click and "Copy link location" and paste that into the address bar.

**Comment by atopal** [comment]
I can't reproduce this. Any add-ons that might be interfering?

**Comment by peterbe** [comment]
It's still a problem today. Am I the only one?! 
For the record, I haven't been able to reproduce it on http://localhost.org:8000/en-US/docs/MDN/Jobs
But, I haven't run it in production mode. Yet.

Perhaps we just stall on this and wait till other people have the same problem. If they do. 
I know we still have some JavaScript event that listens for any click on the `<body>` and all its descendants, but I can't yet see how it could affect me. And why is it only me?!?!

**Comment by peterbe** [comment]
My FirefoxNightly has had many updates since I first filed this. BUT IT STILL DOESN'T WORK! So weird. 
Am I the only one suffering from this strange bug?

**Comment by tobinmori** [comment]
Limited to Peter only? Close for now, until further reports.
